packages/fxa-content-server/tests/functional/delete_account.js

```diff
@@ -51,10 +54,41 @@ registerSuite('delete_account', {
               selectors.SETTINGS_DELETE_ACCOUNT.DETAILS
             )
           )
-          .then(fillOutDeleteAccount(PASSWORD))
+          .findAllByCssSelector(selectors.SETTINGS_DELETE_ACCOUNT.CHECKBOXES)
+          .then(checkboxes => checkboxes.map(checkbox => checkbox.click()))
+          .end()
+          .then(
+            type(selectors.SETTINGS_DELETE_ACCOUNT.INPUT_PASSWORD, 'PASSWORD')
```

Comment:
I would probably call the `'PASSWORD'` something like `invalid password` or something similar.

Reply:
I made the suggested change.